specs/protocol/superchain-config.md

```diff
@@ -5,60 +5,243 @@
 **Table of Contents**
 
 - [Overview](#overview)
-- [Configurable values](#configurable-values)
-- [Configuration data flow](#configuration-data-flow)
-  - [Pausability](#pausability)
-    - [Paused identifiers](#paused-identifiers)
-    - [Scope of pausability](#scope-of-pausability)
+- [Definitions](#definitions)
+  - [Guardian](#guardian)
+  - [Pause Deputy](#pause-deputy)
+  - [Pause Identifier](#pause-identifier)
+  - [Withdrawal Liveness](#withdrawal-liveness)
+  - [Withdrawal Safety](#withdrawal-safety)
+  - [Pause Mechanism](#pause-mechanism)
+- [Invariants](#invariants)
+  - [iSUPC-001: The Guardian can only cause a Withdrawal Liveness failure](#isupc-001-the-guardian-can-only-cause-a-withdrawal-liveness-failure)
+    - [Impact](#impact)
+  - [iSUPC-002: The Pause Deputy can only cause a temporary Withdrawal Liveness failure](#isupc-002-the-pause-deputy-can-only-cause-a-temporary-withdrawal-liveness-failure)
```

Comment:
Is it intentional that the guardian "can only cause a **_[permanent]_** Withdrawal Liveness failure" but the pause deputy "can only cause a **_temporary_** Withdrawal Liveness failure"?

Reply:
Yes this is intentional. The Guardian (i.e., a majority of the security council) can cause a permanent liveness failure if they choose to.